channeldb/graph.go

```diff
@@ -190,6 +190,36 @@ func (c *ChannelGraph) Database() *DB {
 	return c.db
 }
 
+// NumNodes returns the number of nodes currently in the graph
+func (c *ChannelGraph) NumNodes(tx *bbolt.Tx) (int, error) {
```

Comment:
I would add to each commit message the resulting performance impact of the change.

Reply:
Or at least why the change has a positive impact.